src/Processors/QueryPlan/WindowStep.h

```diff
@@ -24,9 +24,9 @@ class WindowStep : public ITransformingStep
 
     void describeActions(JSONBuilder::JSONMap & map) const override;
     void describeActions(FormatSettings & settings) const override;
+    WindowDescription window_description;
```

Comment:
It's not used anywhere so no reason to make it public.

Reply:
Done.